elgg-plugin.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Friend Request
+**/
+return [
+	'bootstrap' => \ColdTrick\FriendRequest\Bootstrap::class,
+	'actions' => [
+		'friends/add' => [
+			'controller' => \ColdTrick\FriendRequest\ActionFriendsAdd::class,
+		],
+		'friends/remove' => [
+			'controller' => \ColdTrick\FriendRequest\ActionFriendsRemove::class,
+		],
+		'friend_request/approve' => [
+			'controller' => \ColdTrick\FriendRequest\ActionFriendsApprove::class,
+		],
+		'friend_request/decline' => [
+			'controller' => \ColdTrick\FriendRequest\ActionFriendsDecline::class,
+		],
+		'friend_request/revoke' => [
+			'controller' => \ColdTrick\FriendRequest\ActionFriendsRevoke::class,
+		],
+	],
+	'routes' => [
+		'view:friend_request' => [
```

Comment:
if believe to correct naming conventions is something like `collection:user:user:friend_request`. Since it's a listing of users